mobile/src/main/res/layout-v21/item_vault_files.xml

```diff
@@ -16,6 +16,7 @@
         app:layout_constraintEnd_toStartOf="@id/imagesBtn"
         app:layout_constraintTop_toTopOf="parent"
         app:mainTitleText="@string/Vault_AllFiles_Title"
+        android:contentDescription="@string/action_open_all_files"
         app:rightImg="@drawable/ic_myfiles"
```

Comment:
are you sure this is working? you would typically need create a custom namespace and define the attributes within it.

Reply:
also make we have two layouts here `layout-v21/item_vault_files.xm` and` layout/item_vault_files.xm` make sure to add the attributes to both files!